src/shared/components/idme/idme.js

```diff
@@ -4,18 +4,17 @@ import troopImage from 'images/Troop.png';
 import styles from './idme.css';
 
 class Idme extends Component {
-  componentWillMount() {
-
-  }
+  componentWillMount() {}
```

Comment:
I was gonna say you should remove this, but hell we should just ditch the Pure component here and go SFC.